x11-packages/tilix/build.sh

```diff
@@ -0,0 +1,26 @@
+TERMUX_PKG_HOMEPAGE=https://gnunn1.github.io/tilix-web
+TERMUX_PKG_DESCRIPTION="A tiling terminal emulator for Linux using GTK+ 3"
+TERMUX_PKG_LICENSE="MPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_VERSION=1.9.6
```

Comment:
If you want to embed the GtkD build in `tilix`'s build please provide a version for both.

```suggestion

TERMUX_PKG_VERSION=(
	1.9.6 # tilix
	3.1.0 # gtkd
)
```

Reply:
(used to apply patch to trim soversion)